packages/lesswrong/lib/instanceSettings.ts

```diff
@@ -139,6 +139,16 @@ export const taggingNamePluralSetting = {get: () => pluralize(taggingNameSetting
 export const taggingNamePluralCapitalSetting = {get: () => pluralize(startCase(taggingNameSetting.get()))}
 export const taggingNameIsSet = {get: () => taggingNameSetting.get() !== 'tag'}
 
+/** Value for tags in the url schema, if set. This allows the url for tags to
+ * be something other than the tag name, e.g. LessWrong is setting this to "w",
+ * for example www.lesswrong.com/w/tagname.
+ * Defaults to tag setting name.
+ */
+const usePluralTagName = !isLWorAF && taggingNameIsSet.get();
+export const taggingUrlSchemaSetting = new PublicInstanceSetting<string>('taggingUrlSchema', taggingNameSetting.get(), 'optional')
+export const taggingUrlSchemaIsSet = {get: () => taggingUrlSchemaSetting.get() !== taggingNameSetting.get()}
+export const tagUrlBaseSetting = {get: () => `${taggingUrlSchemaSetting.get() || (usePluralTagName ? taggingNamePluralSetting.get() : taggingNameSetting.get())}`}
```

Comment:
I don't quite understand what this is doing

Reply:
taggingUrlSchema optionally lets you set a url base that's not your tag name
we then compute the actual base to be used based on whether or not it's set, and whether or not we're capitalizing

we also have a setting for whether a url base is set to mirror whether tag name is set